ampd-handlers/src/bin/stellar/main.rs

```diff
@@ -0,0 +1,78 @@
+mod error;
+mod handler;
+
+use ampd::stellar::rpc_client::Client;
+use ampd::url::Url;
+use ampd_handlers::tracing::init_tracing;
+use ampd_sdk::config;
+use ampd_sdk::runtime::HandlerRuntime;
+use axelar_wasm_std::chain::ChainName;
+#[cfg(debug_assertions)]
+use dotenv::dotenv;
+use error_stack::{Result, ResultExt};
+use serde::{Deserialize, Serialize};
+use tokio_util::sync::CancellationToken;
+use tracing::Level;
+
+use crate::error::Error;
+use crate::handler::Handler;
+
+#[derive(Debug, Deserialize, Serialize)]
+struct StellarHandlerConfig {
+    #[serde(deserialize_with = "Url::deserialize_sensitive")]
+    rpc_url: Url,
+}
+
+fn build_handler(
+    runtime: &HandlerRuntime,
+    chain_name: ChainName,
+    config: StellarHandlerConfig,
+) -> Result<Handler<Client>, Error> {
+    let rpc_client = Client::new(
+        config.rpc_url,
+        runtime.monitoring_client.clone(),
+        chain_name.clone(),
+    )
+    .change_context(Error::HandlerStart)?;
+
+    let handler = Handler::builder()
+        .verifier(runtime.verifier.clone())
+        .voting_verifier_contract(runtime.contracts.voting_verifier.clone())
+        .chain(chain_name)
+        .rpc_client(rpc_client)
+        .monitoring_client(runtime.monitoring_client.clone())
+        .build();
+
+    Ok(handler)
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Error> {
+    #[cfg(debug_assertions)]
+    dotenv().ok();
```

Comment:
could you use `dotenv_flow().ok();` instead? it has been updated in the evm handler. The advantage of using `dotenv_flow` is that it allows the use of different `.env` files by specifying an ENV variable like so: `DOTENV_ENV=ganache2 cargo run --bin multisig-handler` to use `.env.ganache2` instead of just `.env`